ui/src/components/facets/FacetCard.js

```diff
@@ -1,182 +1,30 @@
-          checked={
-            this.props.selectedValues != null &&
-            this.props.selectedValues.includes(value.name)
-          }
-        />
-        <ListItemText
-          className={classes.facetValueNameAndCount}
-          classes={{ primary: this.isDimmed(value) ? classes.grayText : null }}
-          primary={
-            <div style={{ display: "grid", gridTemplateColumns: "auto 50px" }}>
-              <div className={classes.facetValueName}>{value.name}</div>
-              <div className={classes.facetValueCount}>{value.count}</div>
-            </div>
-          }
-        />
-      </ListItem>
-    ));
-
-    return (
-      <div className={classes.facetCard}>
-        <Typography>{this.props.facet.name}</Typography>
-        {this.props.facet.name != "Samples Overview" ? (
-          <Typography className={classes.totalFacetValueCount}>
-            {this.sumFacetValueCounts(
-              this.props.facet.values,
-              this.props.selectedValues
-            )}
-          </Typography>
-        ) : null}
-        <Typography className={classes.facetDescription}>
-          {this.props.facet.description}
-        </Typography>
-        <List dense className={classes.facetValueList}>
-          {facetValueDivs}
-        </List>
-      </div>
-    );
-  }
-
-  isDimmed(facetValue) {
-    return (
-      this.props.selectedValues != null &&
-      this.props.selectedValues.length > 0 &&
-      !this.props.selectedValues.includes(facetValue.name)
-    );
-  }
-
-  /**
-   * @param facetValues FacetValue[] to sum counts over
-   * @param selectedValueNames Optional string[] to select a subset of facetValues to sum counts for
-   * @return number count the total sum of all facetValue counts, optionally filtered by selectedValueNames
-   * */
-  sumFacetValueCounts(facetValues, selectedValueNames) {
-    let count = 0;
-    if (selectedValueNames == null || selectedValueNames.length === 0) {
-      facetValues.forEach(value => {
-        count += value.count;
-      });
-    } else {
-      facetValues.forEach(value => {
-        if (selectedValueNames.indexOf(value.name) > -1) {
-          count += value.count;
-        }
-      });
-    }
-    return count;
-  }
-
-  onClick(facetValue) {
-    // facetValue is a string, eg "female"
-    let isSelected;
-    if (
-      this.props.selectedValues != null &&
-      this.props.selectedValues.length > 0 &&
-      this.props.selectedValues.includes(facetValue)
-    ) {
-      // User must have unchecked the checkbox.
-      isSelected = false;
-    } else {
-      // User must have checked the checkbox.
-      isSelected = true;
-    }
-
-    this.props.updateFacets(
-      this.props.facet.es_field_name,
-      facetValue,
-      isSelected
-    );
-  }
+import TextFacet from "components/facets/TextFacet";
+import HistogramFacet from "components/facets/HistogramFacet";
+
+function FacetCard(props) {
+  const { classes } = props;
+  const rand = Math.random() < 0.5;
```

Comment:
Delete.

Actually, this class doesn't do much. Delete this file and inline in FacetsGride.js.